zh/middleware/modules_communication.md

```diff
@@ -0,0 +1,117 @@
+# Modules Reference: Communication
```

Comment:
I've noticed that, though it would be better if you add a new English file and add it to the Chinese folders.

As for this translation, I rarely translate the `comment` part or the source code. So I just translated the rest of them